src/CMakeLists.txt

```diff
@@ -1,8 +1,19 @@
 find_package(PkgConfig REQUIRED)
-pkg_check_modules(EventdEvent REQUIRED libeventd-event>=0.3)
+pkg_check_modules(EventdEvent REQUIRED libeventd>=0.3)
 pkg_check_modules(EventdPlugin REQUIRED libeventd-plugin>=0.3)
 pkg_check_modules(SystemdJournal REQUIRED libsystemd-journal)
 
+macro(pkg_config_variable PKG VAR OUT)
+    execute_process(COMMAND ${PKG_CONFIG_EXECUTABLE} --variable=${VAR} ${PKG}
+        OUTPUT_VARIABLE ${OUT})
+    string(STRIP ${${OUT}} ${OUT})
```

Comment:
I read the Autotools manuals, I read part of CMake doc. Autotools are nicer to me, and bring way more strictness that CMake does, because people at least know they can screw thing up easily.
The simple facts that you have to use `GNUInstallDirs` to have a (correct) way to install to *the right place* is relevant. For example, your current code would be wrong for Debian, since they put stuff in `lib/<arch>` while a simple `--libdir` works for Autotools.

Of course there are bad users everywhere, and I saw really bad Autotools usages (worse than the worst CMake one, I agree) but they are probably even older that CMake ones. :-)

At least CMake is not Python…